contracts/renderers/PartyNFTRenderer.sol

```diff
@@ -38,11 +39,118 @@ contract PartyNFTRenderer is RendererBase {
         royaltyAmount = 0;
     }
 
+    function contractURI() external view override returns (string memory) {
+        // Check if party has a custom token URI set.
+        {
+            MetadataRegistry metadataRegistry = MetadataRegistry(
+                _GLOBALS.getAddress(LibGlobals.GLOBAL_METADATA_REGISTRY)
+            );
+
+            (
+                string memory name,
+                string memory description,
+                string memory image,
+                string memory banner
+            ) = metadataRegistry.customPartyContractURIByCrowdfund(
+                    // Get the crowdfund that created this party.
+                    Crowdfund(PartyGovernanceNFT(address(this)).mintAuthority())
+                );
+
+            // If the party has a custom token URI set, return it.
+            if (
+                bytes(name).length > 0 &&
+                bytes(description).length > 0 &&
+                bytes(image).length > 0 &&
+                bytes(banner).length > 0
+            ) {
+                return
+                    string.concat(
+                        "data:application/json;base64,",
+                        Base64.encode(
+                            abi.encodePacked(
+                                '{"name":"',
+                                name,
```

Comment:
Can we include the item id after the name? i.e. `[custom name] #[item id]`

Reply:
This is for the collection name. I assume you mean for the party card name?

Reply:
ah, yes -- my b